### DISCUSSION TITLE
False extrema with

### DISCUSSION THREAD

**Discussion by j-ostiguy** [CliMA/Oceananigans.jl#3095] [Computational science!]
Hi, 

Starting with the ocean wind mixing and turbulence example - I noticed that if I set the initial temperature to be uniform, the temperature increases slightly after some time. I changed the advection scheme to WENO (and the closure to SmagorinskyLilly) which improved the results but did not completely resolve the issue. There is only surface cooling, and so I'm not sure why this is occurring. 

Here is a figure where the initial temperature is 5 and after 60 minutes, T reaches 5.0011. 

![temp_Pr10_WENO_60mins](https://user-images.githubusercontent.com/123963410/235699860-cc2f6a76-b152-45db-ac19-9df2bb6cb455.png)

**Comment by glwagner** [comment]
The very small spurious extrema in temperature is an ordinary oscillatory error associated with finite volume numerical advection. WENO reduces spurious extrema but cannot eliminate them. You can try to fix the bounds of temperature if you like (see the docstring for WENO on how to do that). Note though that our bounds preserving WENO scheme is also approximate. Do you require strictly bounds preserving numerics?

**Comment by navidcy** [comment]
Could we change the title of this discussion to sometime more descriptive? Seems like an unfinished sentence?
